cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -4,9 +4,9 @@
 import os.path
 import pprint
 import traceback
-import urllib
-import urllib2
-import urlparse
+import urllib.request, urllib.parse, urllib.error
+import urllib.request, urllib.error, urllib.parse
+import urllib.parse
```

Ruff (0.8.2)

Redefinition of unused `urllib` from line 8

Remove definition: `urllib`

(F811)